src/I18n/languages/de.json

```diff
@@ -70,6 +71,8 @@
     "reportError": "Fehler Melden"
   },
   "authentication": {
+    "noTenantFoundTitle" : "No organization found",
+    "noTenantFoundMessage" : "Do you want to add a new organization using a QR-Code?",
```

Comment:
Almost but in my last comment it was:
Do you want to register a new organization using a QR-Code?

![image](https://user-images.githubusercontent.com/25266945/107048710-8907a880-67c9-11eb-8bbe-864f7d85e05f.png)

Reply:
Ok I was not sure about the true meaning or "register", it is fixed

Reply:
Fixed